llvm/lib/Target/AIE/AIELegalizerInfo.cpp

```diff
@@ -551,44 +551,46 @@ bool AIELegalizerInfo::legalizeCustom(LegalizerHelper &Helper,
   llvm_unreachable("Un-expected custom legalization");
 }
 
-bool AIELegalizerInfo::pack32BitVector(LegalizerHelper &Helper,
-                                       MachineInstr &MI,
-                                       Register SourceReg) const {
+bool AIELegalizerInfo::packBitVector(LegalizerHelper &Helper, MachineInstr &MI,
+                                     Register SourceReg) const {
```

Comment:
Nit: `SourceReg` is an artifact of having a single function for packing/unpacking. Since we are anyway passing `MI` and access `DstReg` in the function, I'd suggest to remove this parameter to make the code easier to understand. 
For `packBitVector`, we have `SourceReg` == `DstReg` anyway

Reply:
Yeah, that makes sense. I changed it, thanks for the review.